selenium/src/test/java/com/epam/tc/hw2/ex1/Exercise1Test.java

```diff
@@ -0,0 +1,102 @@
+
+        new WebDriverWait(driver, Duration.ofSeconds(5))
+                .until(ExpectedConditions.presenceOfElementLocated(By.cssSelector("li.uui-profile-menu"))).click();
+        driver.findElement(By.id("name")).sendKeys("Roman");
+        driver.findElement(By.id("password")).sendKeys("Jdi1234");
+        driver.findElement(By.id("login-button")).click();
+
+        /* Assert Username is loggined.*/
+        softly.assertThat(driver.findElement(By.id("user-name")).getText()).as("Roman Iovlev is not logged")
+                .isEqualTo("ROMAN IOVLEV");
+
+        /* Assert that there are 4 items on the header section are displayed and they have proper texts. */
+        List<WebElement> items = driver.findElements(By.cssSelector(".uui-navigation.nav.navbar-nav.m-l8>li"));
+        softly.assertThat(items.size() == ITEMS_IN_HEADER).as("Incorrect number of items").isTrue();
+
+        for (int i = 0; i < items.size(); i++) {
+            softly.assertThat(items.get(i).isDisplayed()).as("Element is not displayed");
+        }
+        List<String> expectedHeaderItems = List.of("HOME", "CONTACT FORM", "SERVICE", "METALS & COLORS");
+        for (int i = 0; i < items.size(); i++) {
+            softly.assertThat(items.get(i).getText()).as("Text is incorrect").isEqualTo(expectedHeaderItems
+                    .get(i));
+        }
+
+        /* Assert that there are 4 images on the Index Page and they are displayed.*/
+        List<WebElement> icons = driver.findElements(By.className("benefit-icon"));
+        softly.assertThat(icons.size() == NUMBER_OF_IMAGES).as("Incorrect number of images")
+                .isTrue();
+        for (int i = 0; i < icons.size(); i++) {
+            softly.assertThat(items.get(i).isDisplayed()).as("Image is displayed");
+        }
+
+        /* Assert that there are 4 texts on the Index Page under icons and they have proper text. */
+        List<String> listOfTextUnderImages = List.of("To include good practices\n"
+                + "and ideas from successful\n" + "EPAM project", "To be flexible and\n"
+                + "customizable", "To be multiplatform", "Already have good base\n"
+                + "(about 20 internal and\n" + "some external projects),\n" + "wish to get more…");
+        List<WebElement> textElementsUnderImages = driver.findElements(By.className("benefit-txt"));
+
+        for (int i = 0; i < textElementsUnderImages.size(); i++) {
+            softly.assertThat(textElementsUnderImages.get(i).getText()).as("Text is incorrect")
+                    .isEqualTo(listOfTextUnderImages.get(i));
+        }
+
+        for (int i = 0; i < textElementsUnderImages.size(); i++) {
+            softly.assertThat(textElementsUnderImages.get(i).isDisplayed()).as("Text is not displayed");
+        }
+
+        /* Assert that there is the iframe with “Frame Button” exist. */
+        driver.findElement(By.id("frame"));
```

Comment:
`driver.findElement(...)` is not assigned here so it's purpose is not very clear;
Let's assign it to a variable so that we can reuse it? for example:
```java
WebElement frame = driver.findElement(By.id("frame"));
softly.assertThat(frame.isDisplayed()).as("There is no button");
```